locust/stats.py

```diff
@@ -694,18 +696,19 @@ def print_stats(stats, current=True):
 
 def print_percentile_stats(stats):
     console_logger.info("Percentage of the requests completed within given times")
-    console_logger.info((" %-" + str(STATS_NAME_WIDTH) + "s %8s %6s %6s %6s %6s %6s %6s %6s %6s %6s %6s %6s") % (
-        'Name', 
-        '# reqs', 
-        '50%', 
-        '66%', 
-        '75%', 
-        '80%', 
-        '90%', 
-        '95%', 
-        '98%', 
-        '99%', 
-        '99.9%', 
+    console_logger.info((" %-" + str(STATS_TYPE_WIDTH) + "s %-" + str(STATS_NAME_WIDTH) + "s %8s %6s %6s %6s %6s %6s %6s %6s %6s %6s %6s %6s") % (
+        'Type',
```

Comment:
So `type` is passed in as `method`, and that's what we want the title to be?

Reply:
Yes Type is the column name in the CSV. Write now we write HTTP method to it, but i think we want to keep the column name generic.
Also in the past 1 file has method as column name and other file did not have separate column.